### PR TITLE
Move least needed file_engine binaries to release, named 'all_os_file_engines'

### PR DESCRIPTION
**Pull request by cometsong**

### CODE REVIEW

**Comment by cometsong** [comment]
#225 
@Konfekt - the 'doc' change look suitable as notification for this change?
The 'bin' directory is changed from 39M to 12M.

File: doc/ctrlspace.txt [hunk 1]
```diff
@@ -1443,6 +1443,12 @@ pure Vim Script (like `CtrlP`) thus it affects the performance too.
 
 General Assumptions~
 
+The file engine binaries have been compiled for various OS's and cpu
+types.  The programs available in the primary repository include Linux,
```
Comment (Konfekt) [inline doc/ctrlspace.txt]: ... CPU types, but only those for Linux, MacOS and Windows under 64 bit architectures are available in the git repository. The versions for their 32 bit architecture counterparts, as well as for the FreeBSD, NetBSD and OpenBSD under 32 and 64 bit architectures can still be dowloaded ...

**Comment by Konfekt** [comment]
Excellent work. I proposed a more exact reformulation of the middle phrase.

**Comment by cometsong** [comment]
Done and done.

**Pull request merged**